docs/DeveloperGuide.md

```diff
@@ -444,3 +444,18 @@ testers are expected to do more *exploratory* testing.
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Appendix: Planned Enhancements
+Team Size: 5
+
+1. Adding appointments that span multiple days: Currently, to add appointments that span multiple days, you can add multiple consecutive appointments for each day to cover the entire period. However, this process is tedious and time-consuming. We plan to support multi-day appointments, allowing users to add them in one step.
+2. Imposing constraints on archived people: Currently, archived people and their appointments can still be modified, and these appointments are still displayed in the appointment list and calendar. We plan to restrict this by preventing modifications to archived people and hiding/showing their appointments only when required, similar to how list works. For example, to view archived people, you can do list archive/. 
+3. Enhancing duplicate detection: Currently, duplicates are taken as persons with the exact same full name. We plan to enhance this by disallowing other details such as phone numbers and emails to be duplicated as these are unique too and should not be allowed to be repeated. 
+4. Enhancing validation checks : Currently, the validation checks for some fields are not very strict. For instance, the address can be anything that is not blank. This means the same address is treated as different if the user input differs by even 1 whitespace. Therefore, we plan to improve on validation checks for fields like this to minimise the possibilities of user errors like adding a single whitespace and ensure standardization of the representation of these fields. 
+5. Modifying available schemes: Currently, SocialBook has two available schemes that a person can be eligible for. We plan to allow the user to add more schemes or delete existing schemes into SocialBook that could be applicable for persons they are dealing with, so that they can delete schemes that are outdated or add new schemes that are created in future. 
+6. Viewing the details of the different schemes. Currently, it is difficult for users to know what are the requirements for a person to enroll under a scheme and to find out the exact details of the scheme such as the benefits. We plan to allow the user to view these details about available schemes in SocialBook so that the details are easily retrievable. 
```

Comment:
Should be ":". Also should use "eligible" instead of "enroll" since thats the term we use throughout.

Reply:
Thanks for spotting this error!